cypress/e2e/_app.test.ts

```diff
@@ -0,0 +1,59 @@
+declare global {
+    interface Window {
+        dataLayer?: {
+            event: {}
+        }[]
+    }
+}
+
+beforeEach(() => {
+    cy.visit('/')
```

Comment:
TypeScript is flagging `cy`. I may be missing where this is typed?

Reply:
You may have to install dependencies to get the types from not warning you 🤓